sdk/python/kfp/compiler/compiler.py

```diff
@@ -677,6 +800,9 @@ def compile(self, pipeline_func, package_path, type_check=True):
       yaml.Dumper.ignore_aliases = lambda *args : True
       yaml_text = yaml.dump(workflow, default_flow_style=False)
 
+      if package_path is None:
```

Comment:
What are the use cases where the YAML text is need?
The `Compiler()._compile` method returns the workflow dict which seems more useful.

Reply:
i was using it for visual debugging while i was developing.  i figured why not leave the option for anyone else who ends up working on the compiler.  what do you guys think?